portfolio.html

```diff
@@ -8,6 +8,259 @@
     <link href="portfolio.css" rel="stylesheet">
 </head>
 <body>
+
+    <div class="desktop-background-images">
+    </div>
+
+    <!-- Desktop version header -->
+    <header class="header">
+        <ul class="menu">
+            <li class="menu menu-items1"><a href="#"></a>Portfolio</li>
+            <li class="menu menu-items2"><a href="#"></a>About</li>
+            <li class="menu menu-items3"><a href="#"></a>Contact</li>
+        </ul>
+    </header>
+
+    <!-- Desktop version headline -->
+    <section class="desktop-headline">
+        <div class="desktop-headline-first-text">
+            <h2 class="desktop-headline-heading">Hey there. I’m Norbi<br><span>I’m a software developer</span></h2>
+        </div>
+
+        <div class="desktop-headline-supporting-text">
+            <p class="desktop-headline-secondary-text">
+                I can help you build a product , feature or website Look through some of my work and <br> experience! If you like what you see and have a project you need coded, <br>
+                don’t hestiate to contact me.
+            </p>
+        </div>
+
+        <ul class="desktop-social-media-icons">
+            <li><a href="https://github.com/norbiboy1"><img src="./images/NormalButton2/Tertiary/Icons/github.png" class="github" alt="Github icon"></a></li>
+            <li><a href="https://www.linkedin.com/in/siko-norbert/"><img src="./images/NormalButton2/Tertiary/Icons/linkedin.png" class="linkedin" alt="Linkedin icon"></a></li>
+            <li><a href="#"><img src="./images/NormalButton2/Tertiary/Icons/angelis.png" class="angelis" alt="Angelis icon"></a></li>
+            <li><a href="https://twitter.com/NorbertSiko"><img src="./images/NormalButton2/Tertiary/Icons/twitter.png" class="twitter" alt="Twitter icon"></a></li>
+            <li><a href="#"><img src="./images/NormalButton2/Tertiary/Icons/Vector.png" class="vector" alt="Vector icon"></a></li>
+        </ul>
+
+        <div class="desktop-normal-button">
+            <button type="button" class="btn2">
+                <img src="./images/NormalButton2/Secondary/Icons/Union.png" class="desktop-disabled-button" alt="Arrow down icon">
+            </button>
+        </div>
+    </section>
+
+    <!-- Desktop version work section -->
+    <section class="desktop-work">
```

Comment:
- [ ] It appears that after the new changes were introduced the view of this section is distorted. I am viewing the project at 100% zoom level. Kindly center this section so that it matches the chosen Figma design.

**How I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/187041273-6a44ad6d-1e3a-430d-832b-7c756475d8d8.png)

**The desired design:**

![image](https://user-images.githubusercontent.com/86318284/187041314-871e3472-a825-4a5d-8436-9224c825a40e.png)